python_flask_jwt.py

```diff
@@ -6,7 +6,10 @@
 import uuid
 import jwt
 import urllib
```

Comment:
For user examples it's always tricky just providing version changes... def not saying this is the RIGHT way just a couple other options. 

```
import sys
if sys.version_info.major > 2:  # Python 3 or later
    from urllib.parse import quote
else:  # Python 2
    from urllib import quote
```
use pip install six 
since the code already has pip dependency for jwt

```
from six.moves.urllib.parse import quote
```
#Then change line 39 to
```
quote(return_to)
```

If not I would make the python3 version the default.